src/Common/DeprecationTrait.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * Copyright © 2017-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace ArchitectureSniffer\Common;
+
+use PHPMD\Node\MethodNode;
+
+trait DeprecationTrait
+{
+
+    /**
+     * @var string
+     */
+    protected $regexp = '(@([a-z_][a-z0-9_]+))i';
```

Comment:
Wouldnt it suffice to look for specific `@deprecated\w` here? no need to parse all possible @ tags :)